src/views/userOp/UserOpLogs.tsx

```diff
@@ -0,0 +1,154 @@
+                <div className="my-4  flex flex-col gap-[2rem]">
+
+                    {logs.map((log, index) => (
+                        <div key={index} className="bg-white shadow rounded-lg p-4">
+                            <div className="flex flex-col gap-4">
+
+                                <div className="flex flex-row md:gap-[2.2rem] max-md:flex-col">
+                                    <strong className='md:w-[8%] '>Address:</strong>
+
+
+                                    <Link href={`https://etherscan.io/address/${log.address}`} target='_blank'>
+                                        <div className="text-indigo-600 hover:text-indigo-800 w-full px-1 py-1">{log.address}</div>
+                                    </Link>
+
+                                </div>
+
+                                <div className="flex flex-row gap-[2rem] max-md:flex-col">
+                                    <strong className='md:w-[8%]  '>Event Name:</strong>
+                                    <span className="rounded px-2 py-1 md:w-[80%] text-gray-500">
+                                        {log.event_name || 'Unknown Event'}
+                                    </span>
+                                </div>
+
+                                <div className="flex flex-row gap-[2rem] max-md:flex-col">
+                                    <strong className='md:w-[8%] '>Topics:</strong>
+                                    <div className="flex flex-col gap-2md: w-[80%]">
+                                        {log.topicsDecoded ? (
+                                            Object.entries(log.topicsDecoded).map(([key, value]) => (
+                                                <div key={key} className=" rounded px-1 py-1 flex flex-row gap-[5px]">
+                                                    <div className='bg-gray-100 px-[7px] py-[1px] rounded-[3px] text-[12px]'>{key}</div>  {value}
+                                                </div>
+                                            ))
+                                        ) : (
+                                            log.topics.map((topic, index) => (
+                                                <div key={index} className=" rounded px-2 py-1">
+                                                    {index}: {topic}
+                                                </div>
+                                            ))
+                                        )}
+                                    </div>
+                                </div>
+
+                                <div className="flex flex-row gap-[2rem] max-md:flex-col">
+                                    <strong className='md:w-[8%]'>Data:</strong>
+                                    <div className="bg-gray-100 rounded px-2 py-1 md:w-[80%]">
+                                        {log.dataDecoded ? (
+                                            <div className="flex flex-col gap-2">
```

Comment:
can we return none if dataDecoded object is present and empty ? 

Check the third object here - https://jiffyscan-frontend-8hrzh4j55-jiffy-labs.vercel.app/userOpHash/0x3c2fa520f6eda90cf51eed4d14a8c2acd35c703cc542f49366abb66d2a7b155e?network=matic